docs/content/quickstart/porter-yaml.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Examine porter.yaml
+description: Examining the Porter YAML configuration
+---
+
+Let's look at one of the key components of a bundle - the manifest file that is created with `porter create` in `porter.yaml`.
+
+```yaml
+
+name: HELLO
+version: 0.1.0
+description: "An example Porter configuration"
+tag: getporter/porter-hello:v0.1.0
+
+mixins:
+  - exec
+
+install:
+  - exec:
+      description: "Install Hello World"
+      command: ./helpers.sh
+      arguments:
+        - install
+
+upgrade:
+  - exec:
+      description: "World 2.0"
+      command: ./helpers.sh
+      arguments:
+        - upgrade
+
+uninstall:
+  - exec:
+      description: "Uninstall Hello World"
+      command: ./helpers.sh
+      arguments:
+        - uninstall
+```
+
+This example code is created directly after running `porter create` and should be modified and customized for your needs. These are not the only configuration options, but let's talk through this example.  
+
+At the top, the bundle's metadata is defined:
+
+```yaml
+
```

Comment:
I noticed all the code blocks are starting with an empty line which is coming through in the final doc. Is that intentional?